src/content/reference/docker-compose.mdx

```diff
@@ -194,6 +194,41 @@ environment:
   DB_HOST: postgres://${DB_HOST:-db}:${DB_PASSWORD}@postgres:5432/postgres
 ```
 
+#### healthcheck (object, optional)
+
+healthcheck declares a check that's run to determine whether or not containers for this
+service are "healthy". 
+
```

Comment:
Also, it's a little confusing that we're using `int` and `string` for some fields and `duration` for others. Should we change `duration` to `int` too for consistency?

Reply:
It's how docke compose spec works. I also prefer to have everything as int with seconds as default measurement